CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ More detailed changelog and description will come later
 
```

Comment:
Change line 5 to "second" release candidate, then update the date. Then looks good to go.